whylog_vim/gui/vim_commander.py

```diff
@@ -0,0 +1,86 @@
+import vim
+
+
+class VimCommander(object):
+
+    @staticmethod
+    def resize(size):
+        vim.command('resize %s' % size)
+
+    @staticmethod
+    def set_file_type():
+        vim.command('setlocal filetype=whylog')
+
+    @staticmethod
+    def get_column():
+        return int(vim.eval('col(".")'))
+
+    @staticmethod
+    def get_current_window_id():
+        return int(vim.eval('winnr()'))
+
+    @staticmethod
+    def normal(command):
+        vim.command("normal %s" % (command,))
+
+    @staticmethod
+    def go_to_window(window_id):
+        assert window_id is not None
+        vim.command('%dwincmd w' % window_id)
+
+    @staticmethod
+    def split_window(name=None):
```

Comment:
http://effbot.org/zone/default-values.htm
I want to aviod problems like this, becouse I don't know how it works with strings.